spec/lib/data_update_scripts/add_single_resource_admin_role_to_users_with_tech_admin_spec.rb

```diff
@@ -4,25 +4,35 @@
 )
 
 describe DataUpdateScripts::AddSingleResourceAdminRoleToUsersWithTechAdmin do
-  let!(:tech_admin1) { create(:user, :tech_admin) }
-  let!(:tech_admin2) { create(:user, :tech_admin) }
   let!(:admin) { create(:user, :admin) }
 
-  it "adds single_resource_admin roles to users with tech_admin roles" do
-    described_class.new.run
-
-    expect(tech_admin1.reload.roles.pluck(:name)).to include("single_resource_admin")
-    expect(tech_admin2.reload.roles.pluck(:name)).to include("single_resource_admin")
+  context "without any tech_admin users" do
+    it "does not add any roles to any other users with roles" do
```

Comment:
Threw in a test just for the heck of it for some extra assurance since we have a test set up for this script 🙃

Reply:
I was really hoping that you would, so thank you!!